src/common/config_parser.cpp

```diff
@@ -194,6 +194,14 @@ void ConfigParser::addOptionsModel(cli::CLIWrapper& cli) {
   cli.add<int>("--dim-emb",
       "Size of embedding vector",
       512);
+  cli.add<int>("--factors-dim-emb",
+      "Embedding dimension of the factors. Only used if concat is selected as factors combining form");
+  cli.add<std::string>("--factors-combine", 
+    "How to combine the factors and lemma embeddings. Options available: sum, concat",
+    "sum");   
```

Comment:
Since concatenation was implemented, two new options were created. One (--factors-combine) that controls if we want to embed the lemmas and factors by summing them (the default option), or apply concatenation.
If sum is chosen it will follow the already implemented embedding from Frank.
In case you chose concatenation a dimension of that embedding must be specified with --factors-dim-emb.